modules/about-dynamic-plug-ins.adoc

```diff
@@ -0,0 +1,35 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/web_console/dynamic-plug-ins.adoc
+
+:_content-type: CONCEPT
+[id="about-dynamic-plugins_{context}"]
+= About dynamic plug-ins
+
+A dynamic plug-in allows you to add custom pages and other extensions to your interface at runtime. Plug-ins are registered with the console using the `ConsolePlugin` custom resource, and they are enabled in the console operator configuration by a cluster administrator.
+
+[id="dynamic-plugins-features"]
+== Key features
+
+A dynamic plug-in will allow you to make customizations such as:
+
+* Add custom pages
+* Add perspectives and update navigation items
+* Add tabs and actions to resource pages
+
+== PatternFly 4 Guidelines
+When creating your plug-in, follow these guidelines for using PatternFly:
+
+* Use link:https://www.patternfly.org/v4/[PatternFly4] components. Core PatternFly components are available through the SDK.
+* Use PatternFly components and PatternFly CSS variables. Using PatternFly components and variables will help your plug-in look consistent in future console versions.
```

Comment:
Should we combine these two bullets? (Since they both say to use PF components?)